backend/services/documents_service.py

```diff
@@ -0,0 +1,155 @@
+from typing import Any, Dict, Tuple, cast
+
+from backend.document_status import Status
+from backend.mongodb_handler import mongo
+from backend.role import Role
+
+from .send_email import send_email
+
+
+def create_document(body: Dict, user_id: str) -> Tuple[Any, int]:
+    if not body.get("document_name", False):
+        return body, 400
+
+    document_name = body["document_name"]
+    document_identifier = mongo.create_document(document_name, user_id)
```

Comment:
too long :) just `document_id` or `doc_id`. Not really necessary to use so much symbols for variable names